spotipy/util.py

```diff
@@ -35,21 +35,28 @@ def prompt_for_user_token(
         "    spotipy.Spotify(auth_manager=auth_manager)",
         DeprecationWarning
     )
-    """ prompts the user to login if necessary and returns
-        the user token suitable for use with the spotipy.Spotify
-        constructor
-
-        Parameters:
-
-         - username - the Spotify username (optional)
-         - scope - the desired scope of the request (optional)
-         - client_id - the client id of your app (required)
-         - client_secret - the client secret of your app (required)
-         - redirect_uri - the redirect URI of your app (required)
-         - cache_path - path to location to save tokens (optional)
-         - oauth_manager - Oauth manager object (optional)
-         - show_dialog - If true, a login prompt always shows (optional, defaults to False)
-
+    """Prompt the user to login if necessary and return a user token.
+
+        Prompts the user to login if necessary and returns the user token
+        suitable for use with the `spotipy.Spotify` constructor.
+
+        Args:
+            username: The Spotify username. (optional)
+            scope: The desired scope of the request. (optional)
+            client_id: The client ID of your app. (required)
+            client_secret: The client secret of your app. (required)
+            redirect_uri: The redirect URI of your app. (required)
+            cache_path: Path to location to save tokens. (required)
+            oauth_manager: OAuth manager object. (optional)
+            show_dialog: If True, a login prompt always shows.
+                         If not, it will default to False. (optional)
+
+        Returns:
+            A token suitable for use with the `spotipy.Spotify` constructor.
+            None: If no valid token is found.
+
+        Raises:
+            SpotifyException: If no credentials are set.
```

Comment:
Please revert this part, so that the format is the same as in the other docstrings.
(Yes, our format isn't the best either, but converting every docstring to a new format this is a task for v3)

Reply:
I have reverted the part so that it matches the other docstrings in client.py along with some subtle edits to it (formatting).